testing/sdk_upgrade.py

```diff
@@ -147,6 +147,23 @@ def get_config(package_name, service_name):
     return target_config
 
 
+def update_service(package_name, service_name, additional_options=None, to_package_version=None):
```

Comment:
Nice!

Reply:
I am not sure on `tempfile` semantics but by the time the code executes `sdk_cmd.svc_cli.....` isn't the temp file out of scope ? I ruled out this option initially thinking the file would be deleted by then.